app/[id]/location.tsx

```
@@ -1,235 +1,216 @@
-                    <View className="gap-2 my-5">
-                        <View>
-                            <Text className="text-xl font-bold dark:text-white">
-                                Sondage en cours
-                            </Text>
-                            <Text className="text-sm dark:text-gray-200">
-                                Votes pour ton choix d'hébergement préféré
+                    }
+                    ListHeaderComponent={
+                        <View className="gap-2">
+                            <Text className="text-2xl font-bold dark:text-white">
+                                Étapes de l'escapade
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)